tests/integration/data_sources_and_expectations/test_misconfigured_expectations.py

```diff
@@ -0,0 +1,184 @@
+import datetime as dt
+
+import pandas as pd
+import pytest
+
+import great_expectations.expectations as gxe
+from tests.integration.conftest import parameterize_batch_for_data_sources
+from tests.integration.data_sources_and_expectations.test_canonical_expectations import (
+    ALL_DATA_SOURCES,
+)
+from tests.integration.test_utils.data_source_config.base import DataSourceTestConfig
+
+PANDAS_DATA_SOURCES: list[DataSourceTestConfig] = [
+    ds for ds in ALL_DATA_SOURCES if ds.label.startswith("pandas")
+]
+SPARK_DATA_SOURCES: list[DataSourceTestConfig] = [
+    ds for ds in ALL_DATA_SOURCES if ds.label.startswith("spark")
+]
+SQL_DATA_SOURCES: list[DataSourceTestConfig] = [
+    ds for ds in ALL_DATA_SOURCES if ds not in PANDAS_DATA_SOURCES and ds not in SPARK_DATA_SOURCES
+]
+
+
+@pytest.mark.unit
+def test_parameterization():
+    # Ensure that all data sources are covered
+    assert len(PANDAS_DATA_SOURCES) == 2
+    assert len(SPARK_DATA_SOURCES) == 1
+    assert len(SQL_DATA_SOURCES) == 7
+    assert len(PANDAS_DATA_SOURCES) + len(SPARK_DATA_SOURCES) + len(SQL_DATA_SOURCES) == len(
+        ALL_DATA_SOURCES
+    )
+
+
+class TestNumericExpectationAgainstStrDataMisconfiguration:
+    # Currently bugs with the following (not raising misconfiguration errors at all):
+    #  - sqlite
+    #  - databricks
+    #  - mysql
+    #  - spark-filesystem-csv
```

Comment:
These do not raise any errors (fail on `assert not result.success`) - going to write tickets for each of them to investigate

Reply:
yikes! glad you caught it!